featuretools/primitives/base/aggregation_primitive_base.py

```diff
@@ -80,7 +80,7 @@ def make_agg_primitive(function, input_types, return_type, name=None,
                        stack_on_exclude=None, base_of=None,
                        base_of_exclude=None, description='A custom primitive',
                        cls_attributes=None, uses_calc_time=False,
-                       commutative=False):
+                       commutative=False, number_output_features_keyword=None):
```

Comment:
what if the number of output features isn't dynamic. can the user simply say "there are 4 output features for this primitive"? maybe there needs to be a `number_output_features` parameter too that gets checked before `number_output_features_keyword`